source/reference/api/javascript-sdk.rst

```diff
@@ -20,8 +23,21 @@ The JavaScript SDK have the following npm package names:
 * `@craftercms/search <https://www.npmjs.com/package/@craftercms/search>`__ - Contains tools for integrating your application with Crafter Search.
 * `@craftercms/utils <https://www.npmjs.com/package/@craftercms/utils>`__ - Contains various utilities useful when developing with CrafterCMS
 * `@craftercms/ice <https://www.npmjs.com/package/@craftercms/ice>`__ - Contains JavaScript utilities to use CrafterCMS In Context Editing in your Apps and Sites
-* `@craftercms/experience-builder <https://www.npmjs.com/package/@craftercms/experience-builder>`__ - Contains JavaScript utilities to use CrafterCMS experience builder in your Apps and Sites
+
+The `Video Center Blueprint <https://craftercms.com/marketplace/video-center-blueprint>`__ in the *Public Marketplace* is an example of using the above JavaScript SDK NPM packages.
+
+**Authoring**
+
+* `@craftercms/experience-builder <https://www.npmjs.com/package/@craftercms/experience-builder>`__ - Contains JavaScript utilities to integrate Experience Builder in your Apps and Sites
 * `@craftercms/studio-ui <https://www.npmjs.com/package/@craftercms/studio-ui>`__ - Contains tools to enable building CrafterCMS extensions using all of its native UI components, utils and APIs
 
+The `Authoring Plugin Examples <https://github.com/craftercms/authoring-ui-plugin-examples>`__ repo showcases how to use the above JavaScript SDK packages to build Authoring Plugins.
+
+Compatibility with CrafterCMS Version
+=========================================
+
+As of version 4.0.3, CrafterCMS NPM libraries adhere to a product versioning scheme aligned with the platform version.
```

Comment:
4.3.0

Reply:
This first paragraph is when we started product versioning (in 4.0.3.) When we break from it is the next paragraph.